packages/collaboration-extension/src/collaboration.ts

```diff
@@ -189,3 +191,63 @@ export const userEditorCursors: JupyterFrontEndPlugin<void> = {
     });
   }
 };
+
+export const notebookCellExecutor: JupyterFrontEndPlugin<INotebookCellExecutor> =
```

Comment:
Something to consider: do we want this plugin in a separate package in the future so that when we split frontend and backend (https://github.com/jupyterlab/jupyter-collaboration/issues/269) we can still load this plugin but not the others? This question does not block merging this PR in any way.

Reply:
But does it make sense to have server-side execution without collaboration? The former is based on the latter, so in my opinion they go together.

Reply:
It would be sufficient to have the collaborative drive, right?